ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -219,7 +219,8 @@ def publish_waypoints_ahead(self):
 
         rospy.logwarn("WaypointUpdater-Published speed: {}".format(speeds))
```

Comment:
oh, by the way, can you also change "WaypointUpdater-Published speed" to "wp_updater: published speed"?

Reply:
done.